modules/salt/pages/formulas-suma.adoc

```diff
@@ -0,0 +1,56 @@
+[[formulas-suma]]
+= Formulas Provided by {productname}
+
+Some formulas are provided by default with {productname}.
+Other official formulas can be installed as RPM packages.
+This section contains information about installing formulas provided by {productname} using either the {webui} or RPM packages.
+
+For information about writing custom formulas, see xref:salt:formulas-custom.adoc[].
+
+
+
+== Install Formulas from the {webui}
+
+Formulas provided by {productname} can be installed using the {productname} {webui}.
+
+
+.Procedure: Installing Formulas from the {webui}
+. In the {productname} {webui}, navigate to menu:Systems[List], select the client you want to install the formula on, and navigate to the menu:Systems[Formulas] tab.
+. Check the formula you want to install the selected client.
+. Click btn:[Save].
+. Navigate to the new subtab for the formula, and configure the formula as required.
+. Apply the highstate.
+
+More specific information about each of the {productname} formulas is available in this guide.
+
+
+
+== Install Formulas from an RPM
+
+{suse} releases formulas as RPM packages.
+Available formulas can be located within the [systemitem]``SUSE-Manager-Server-VERSION-Pool`` channel.
```

Comment:
Uyuni variant missing ;)

Reply:
I'm not sure what the Uyuni version is.

Reply:
I see formulas here:

https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-POOL-x86_64-Media1/noarch/

Uyuni resources are without version numbers.